### PR TITLE
update RTPS msg ID's

### PR DESCRIPTION
**Pull request by TSC21**
While we don't find a correct way of auto generating these ID's, we should at least add ID's to all the uORB topics available and its aliases for multi-topics.

### CODE REVIEW

**Comment by TSC21** [comment]
Added a check for the existence or not of the RTPS ID. If there's none, then it throws an error.

File: msg/tools/px_generate_uorb_topic_helper.py [hunk 1]
```diff
@@ -346,11 +346,19 @@ def print_field_def(field):
...
 
+
 def rtps_message_id(msg_id_map, message):
     """
     Get RTPS ID of uORB message
     """
+    msg_id = -1
     for dict in msg_id_map[0]['rtps']:
         if message in dict['msg']:
-            return dict['id']
```
Comment (bkueng) [inline msg/tools/px_generate_uorb_topic_helper.py]: You can keep the return, and when the loop ends just raise the exception.

File: msg/tools/px_generate_uorb_topic_helper.py [hunk 2]
```diff
@@ -346,11 +346,19 @@ def print_field_def(field):
...
-    return 0
+            msg_id = dict['id']
+
+    if msg_id != -1:
+        return msg_id
+    else:
+        raise AssertionError(
+            "%s does not have a RTPS ID set in the definition file. Please add an ID from the available pool!")
```
Comment (bkueng) [inline msg/tools/px_generate_uorb_topic_helper.py]: If I saw this I would ask myself: what's the available pool?
Also do we check for uniqueness of the id?

File: msg/tools/px_generate_uorb_topic_helper.py [hunk 3]
```diff
@@ -346,11 +346,19 @@ def print_field_def(field):
...
+        exit(1)
```
Comment (bkueng) [inline msg/tools/px_generate_uorb_topic_helper.py]: Not needed.

**Comment by TSC21** [comment]
@bkueng I addressed your comments and added a verification for ID clashes and the available pool of ID's.

**Pull request merged**